spaces/S000202/properties/P000126.md

```diff
@@ -0,0 +1,8 @@
+---
+space: S000202
+property: P000126
+value: true
+---
+
+Every set either misses the non-isolated point and is open,
+or contains the non-isolated point and is closed.
```

Comment:
I guess "has a single non-isolated point" could be a utility property on here. It implies door, $T_0$, probably some other low-hanging fruit. I won't add it here though.

Reply:
That's called "almost discrete", but there are variants depending on additional separation axioms or not.  We'll have to be a little careful when introducing this.  But definitely worth having.  It will simplify a lot of things.